volontulo/templates/organizations/organization_offers.html

```diff
@@ -18,7 +18,12 @@
             {% if o.status == 'STAGED' %}
                 <a href="{% url 'offers_view' o.title|slugify o.id %}">Pomóż</a>
             {% else %}
-                <a href="{% url 'activate_offer' o.id %}">Aktywuj</a>
+                <form id="offer_ {{ o.id }}_activate" method="post" action="{% url 'offers_edit' o.title|slugify o.id %}" enctype="multipart/form-data">
```

Comment:
2 problems: 
1) I think it  should be available only for administrator users
2) Like previous - unneded space character